src/helpers/Logging.h

```diff
@@ -0,0 +1,23 @@
+// Joseph Arhar
+
+#ifndef LOGGING_H_
+#define LOGGING_H_
+
+#include <iostream>
+
+// TODO(jarhar): figure out why DEBUG flag isnt working on windows
```

Comment:
The flag is added
`if(NOT CMAKE_BUILD_TYPE MATCHES Release)`
release may be the default so you might need to do something like
`cmake -DCMAKE_BUILD_TYPE=Debug`
when you invoke cmake